src/main/java/com/library/BookRepository.java

```diff
@@ -94,7 +106,7 @@ public List<Book> getBooksFromJsonFile() {
                 JSONObject book = (JSONObject) object; //wskazuję, że obiekt book jest obiektem Json'owym
```

Comment:
te 4 linijki też wynieś do mappera

Reply:
wynikowo ten kawałek chce taki:
for (Object object : booksJson) {
books.add(BookMapper.map((JSONObject) object));
}